src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -821,6 +827,7 @@ def setUp(self):
             # test.  Since the storage is reformatted and the pool metadata is
             # erased when the servers are restarted this check is only needed
             # when the servers are left continually running.
+            self.log_step('setUp(): Destroying any existing pools before the test')
```

Comment:
Again, I don't see this as being a "test step". And in many cases, this doesn't even have to be an assumption of the test environment.